site/en/codelabs/telink-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,739 @@
+ File Download to Flash at address 0x000000: 491700 bytes
+ Total Time: 30087 ms
+```
+
+The flash method for `ot-rcp` is basically the same as that for `ot-cli-ftd`.
+However, please be aware that there are differences in the firmware paths and  names.
+
+After flashing, distinguish the two TLSR9518 development boards by marking them accordingly.
+Label the board flashed with `ot-cli-ftd` as "FTD Joiner" and the board flashed with `ot-rcp` as "RCP".
+
+## Configure a Serial Console for the FTD Joiner Device
+
+Duration: 3:00
+
+To control the FTD Joiner device through the CLI, connect a USB to TTL Serial Convertor Cable to the following pins on the TLSR9518 development board：
+
+| Name  | Pin                 |
+| :---- | ------------------- |
+| RX    | PB3 (pin 15 of J34) |
+| TX    | PB2 (pin 18 of J34) |
+| GND   | GND (pin 23 of J50) |
+
+> aside positive
+>
+> **Note:** Baud rate：115200 bits/s
+
+After connecting the Convertor Cable to the Linux machine, Open PuTTY.
+Then create a new terminal, set the serial port information, and open the serial port.
+
+<img src="img/uart_console.png" alt="uart_console.png" width="624.00" />
+
+ The OpenThread command line reference is here: [OpenThread CLI Reference](https://github.com/openthread/openthread/blob/f7690fe7e9d638341921808cba6a3e695ec0131e/src/cli/README.md).
+ Be sure to prefix with `ot` when using it.
+
+Examples：
+
+```bash
+> ot state
+disabled
+Done
+> ot channel
+11
+Done
+>
+```
+
+## Set up the Raspberry Pi as a OpenThread Border Router
+
+Duration: 9:00
+
+An OpenThread Border Router is a device consisting of two main parts：
+
+* **Raspberry Pi** contains all the services and firmware required to act as a Border Router (BR).
+
+* **RCP** is responsible for Thread communication.
+
+### Radio Co-Processor（RCP）
+
+To flash the `ot-rcp` firmware, follow the same steps as the `ot-cli-ftd` firmware flashing process.
+Connect the TLSR9518 development kit to a USB port on the Raspberry Pi, as depicted in the figure below.
+
+<img src="img/OTBR_overview.png" alt="OTBR_overview.png" width="624.00" />
+
+### Raspberry Pi
+
+1. Ensure that the [Raspbian Bullseye Lite OS image](https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2023-05-03/2023-05-03-raspios-bullseye-armhf-lite.img.xz) or [Raspbian Bullseye with Desktop](https://downloads.raspberrypi.org/raspios_armhf/images/raspios_armhf-2023-05-03/2023-05-03-raspios-bullseye-armhf.img.xz) is properly written to the SD card.
+
+2. You have the option to either SSH onto the Raspberry Pi or work directly with the Raspbian Desktop.
+This Codelab will use SSH.
+
+3. Before proceeding to install OTBR Docker in the next step, make sure to update the local repositories and package manager first.
+
+     ```bash
+     $ sudo apt-get update
+     $ sudp apt-get upgrade
+     ```
+
+### Install Docker
+
+If you just update API, please reboot Raspberry Pi and then open an SSH terminal window.
```

Comment:
What do you mean by "If you just update API"?

Reply:
This is a typographical error where "APT" was written as "API," and it has been corrected in the updated commit.